views/templates/main.twig

```diff
@@ -43,3 +35,8 @@
   var input = {{ jsParams|json_encode|raw }};
 </script>
 <script src="{{ psBaseUri }}modules/autoupgrade/js/upgrade.js"></script>
+
+<script src="//localhost:8080/index.js"></script>
```

Comment:
Mmm, why this `//localhost:8080` address?

Reply:
Because of dev environment, this one needs to be improve too :)